### ISSUE TITLE
`gl track .` says files don't exist when they do

### ISSUE THREAD

**Issue by spderosso** [sdg-mit/gitless#154]
Say you have untracked files `foo` and `bar` in a directory. You can track them with `gl track *` or `gl track foo bar` and that works fine, but if you do `gl track .` you get:

```
✘ Can't track non-existent file ./bar
✘ Can't track non-existent file ./foo
```

**Comment by AndersonReyes** [comment]
Any updates on this? I just got the same error but my files are symlinks on Linux.

**Comment by spderosso** [comment]
Fixed in #220
